src/strands/experimental/bidi/models/openai.py

```
@@ -275,27 +263,16 @@ async def _add_conversation_history(self, messages: Messages) -> None:
 
     async def receive(self) -> AsyncIterable[BidiOutputEvent]:  # type: ignore
         """Receive OpenAI events and convert to Strands TypedEvent format."""
-        # Emit connection start event
```

Changes in this method include:
* Raise runtime if model not "started".
* Remove try/except. Across all model providers, we are going to let exceptions propagate and handle logging centrally in the agent loop.
* Remove yield in finally. In general, it is not appropriate to yield in a finally inside a generator. This can lead to hanging when exceptions are raised.